### ISSUE TITLE
GKE Stackdriver UI: Only showing stdout events

### ISSUE THREAD

**Issue by dieguezz** [googleapis/nodejs-logging-bunyan#123]
Hi, maybe i got this wrong or something, but severity is sent as `INFO` when it should be `ERROR` for this case:

<img width="1465" alt="captura de pantalla 2018-08-16 a las 12 24 01" src="https://user-images.githubusercontent.com/1161455/44203465-4d874880-a14f-11e8-8ab5-6b9c215676a2.png">

Related Issue: https://github.com/googleapis/nodejs-logging-bunyan/issues/59

#### Environment details

  - OS: OSX El Capitan
  - Node.js version: 8.9.0
  - npm version: 6.0.1
  - `@google-cloud/logging-bunyan` version: v0.8.2

#### Steps to reproduce

```
// logger.js
const bunyan = require('bunyan')
const { LoggingBunyan } = require('@google-cloud/logging-bunyan')
const loggingBunyan = new LoggingBunyan()
const logger = bunyan.createLogger({
  name: 'my-service',
  streams: [
    { stream: process.stdout },
    loggingBunyan.stream(),
  ],
})

module.exports = {
  logger,
}

```

```
logger.error('ERROR')
```

**Comment by ofrobots** [comment]
Hi @dieguezz. I see that you have a `stdout` stream in addition to the `LoggingBunyan` stream. For code running on some Google Cloud environments such as GAE and GKE, the stdout is captured by default and sent to Stackdriver Logging (albeit in an unstructured way – since stdout is unstructured) in addition to the stream that `LoggingBunyan` sends.

Are you looking at the `stdout` or the bunyan log in the Stackdriver Logging user interface?

**Comment by dieguezz** [comment]
My env is running on GKE and i am looking at (screenshot) the Stackdriver Logging ui. I am doing different test but i dont find the combination.

**Comment by ofrobots** [comment]
For an AppEngine (GAE) app, here's an example of where the log would shows up:

![image](https://user-images.githubusercontent.com/79017/44228965-802f4200-a14b-11e8-8ed7-8f4fed07d5e1.png)

Let me see how this would show up in a GKE environment.

**Comment by dieguezz** [comment]
You mean this? 
<img width="1398" alt="captura de pantalla 2018-08-16 a las 21 02 02" src="https://user-images.githubusercontent.com/1161455/44229225-ae3a7380-a197-11e8-837d-700f846f95e8.png">

Anyway, all of them appear as `info` unless i add `severity` like: `logger.info('FOO', { severity: 'LEVEL' })`, but that should be done automatically from `level` as stated in related issue
Then appears as should: 
<img width="1362" alt="captura de pantalla 2018-08-16 a las 21 29 02" src="https://user-images.githubusercontent.com/1161455/44230501-9664ee80-a19b-11e8-9ad3-5a8e865cf52d.png">

**Comment by ofrobots** [comment]
@dieguezz those are all logs parsed from `stdout`. The stream sent by `loggingBunyan.stream()` are supposed to show up under a different category in the UI.

I am trying out LoggingBunyan on my own GKE app, and I can reproduce the actual issue that the logs don't seem to be available anywhere in the Logging UI.

I'm investigating this – stay tuned.

**Comment by ofrobots** [comment]
@dieguezz Can you try selecting "All namespace_id" when navigating the logs?

![image](https://user-images.githubusercontent.com/79017/44237062-fb9ced80-a163-11e8-8954-1945c4b477cc.png)

Then you can further filter out the log to see only the bunyan logs:

![image](https://user-images.githubusercontent.com/79017/44237133-3d2d9880-a164-11e8-8083-528847ba032b.png)

Sorry that this is complicated and non-obvious. We will get this improved.

Notes for my fellow maintainers of this repo: The root cause is that this library is sending LogEntries without a `namespace_id` specified in the `container` [monitored resource descriptor](https://cloud.google.com/logging/docs/api/v2/resource-list#resource-types). The Logging UI expects this field to be non-empty. The problem is that the value of `namespace_id` is easily available inside a container – or at least, I don't know the mechanism to get the value. This needs to be investigated and fixed.

**Comment by dieguezz** [comment]
You are right, i see them when i select all. Any ideas of why is this? i'd like to do it selecting just the branch i want (only one namespace)

**Comment by dieguezz** [comment]
Thanks a lot @ofrobots, just one more thing. When are you going to deploy a new release?

**Comment by ofrobots** [comment]
@dieguezz working on a release in #131. Hopefully it can be shipped soon.

**Comment by ofrobots** [comment]
The fix has been released in v0.9.1: https://www.npmjs.com/package/@google-cloud/logging-bunyan?activeTab=versions. @dieguezz please give it a shot and let us know if this works better.

**Comment by elucidsoft** [comment]
This fix needs to be applied to https://github.com/googleapis/nodejs-logging-winston as well as Winston is doing the same thing.
